test/test_tensor_creation_ops.py

```diff
@@ -621,6 +621,29 @@ def test_cat_out(self, device):
         self.assertEqual(a, b)
         self.assertEqual(w[:6], y.view(-1)[:6])
 
+        # Case: 
+        # Reference: https://github.com/pytorch/pytorch/issues/49878
+        for dim in [0, 1]:
+            x = torch.zeros((3, 2), device=device)
+            y = x.narrow(dim, 0, 1)
+
+            if dim == 0:
+                self.assertTrue(y.is_contiguous())
+            else:
+                self.assertFalse(y.is_contiguous())
+
+            val = torch.ones_like(y, device=device) * 3.
```

Comment:
nit: use `full_like` here, and don't hardcode the value to `3` as you are using it again in expected, make a variable for it.